### PR TITLE
Added a new Course under the **Python** section

### PR DESCRIPTION
**Pull request by muhammadanas0716**
I have added a new course under the Python course by ProgrammingWithHarry - it's about 11 hours hours long and includes handwritten notes. Absolutely worth adding.

## What does this PR do?
Added a new Course

## For resources
Youtube Video: https://www.youtube.com/watch?v=EyEqWFvLDT8

### Why is this valuable (or not)?
it is - at least for me and many others it is a worth-watching course.

### How do we know it's really free?
It's on YouTube!

### For book lists, is it a book? For course lists, is it a course? etc.
It's a Course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, with correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.
CHECKLIST COMEPLETE!

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -864,6 +864,7 @@
 * [Python Tutorial - Python for Beginners [Full Course]](https://www.youtube.com/watch?v=_uQrJ0TkZlc) - Moshfegh Hamedani (Programming with Mosh)
 * [Python Tutorials](https://www.youtube.com/playlist?list=PLTjRvDozrdlxj5wgH4qkvwSOdHLOCx10f) - Programming with Mosh (YouTube)
 * [Python Tutorials](https://www.youtube.com/playlist?list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU) - Corey Schafer
+* [Python Tutorial For Beginners (With Notes)](https://www.youtube.com/watch?v=EyEqWFvLDT8) - ProgrammingWithHarry
```
Comment (davorpa) [inline courses/free-courses-en.md:867]: - Not a playlist but +13h video with chapters integrated in the timeline
- Alphabetizing to solve linter warnings

```suggestion
* [Python Tutorial For Beginners (With Notes)](https://www.youtube.com/watch?v=EyEqWFvLDT8) - ProgrammingWithHarry
* [Python Tutorials](https://www.youtube.com/playlist?list=PLTjRvDozrdlxj5wgH4qkvwSOdHLOCx10f) - Programming with Mosh (YouTube)
* [Python Tutorials](https://www.youtube.com/playlist?list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU) - Corey Schafer
```

**Review by davorpa — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (muhammadanas0716) [reply courses/free-courses-en.md]: Done

**Comment by muhammadanas0716** [comment]
I have created another PR.

**Comment by davorpa** [comment]
Duplicate of #7167

**Pull request closed without merging**